finger/views.py

```diff
@@ -29,6 +30,54 @@ def upload_json(request):
         'form': form,
     })
 
+@login_required
+def mail_members(request):
+    if not (request.user.is_staff and request.user.is_superuser):
+        return HttpResponseNotAllowed()
+
+    if request.method == 'POST':
+        form = MailMembersForm(request.POST)
+    else:
+        form = MailMembersForm()
+
+    if form.is_valid():
+        members = form.get_members()
+
+        subject = form.cleaned_data['subject']
+        message = form.cleaned_data['message']
+        sender = request.user.username + '@stacken.kth.se'
+
+        recipients = form.cleaned_data['recipients']
+        if recipients:
+            recipients = [("", recipient) for recipient in recipients.split(',')]
+        else:
+            # todo send to all email addresses
+            recipients = [(f"{member.first_name} {member.last_name}", member.email) for member in members]
+
+        cc = form.cleaned_data['cc']
+        if not cc:
+            cc = "styrelsen@stacken.kth.se"
+
+        context = {
+                'recipients' : recipients,
+                'form' : form,
+                'test' : True,
+        }   
+
+        if request.POST.get("do_it", "") == "true":
+            context['test'] = False
+            to = [email for (_, email) in recipients]
+            to.append(cc)
```

Comment:
Why add a field called "cc" to the TO header? I expect it to end up in the CC header.

Reply:
https://docs.djangoproject.com/en/3.1/topics/email/#django.core.mail.EmailMessage

Reply:
Därför det inte går att CCa med Djangos egna send. Men det har vi redan diskuterat.